src/erc20.rs

```diff
@@ -47,32 +51,40 @@ impl Erc20 {
         .emit();
     }
 
+    /// Returns the name of the token.
     pub fn name(&self) -> String {
         self.name.get_or_default()
     }
 
+    /// Returns the symbol of the token.
     pub fn symbol(&self) -> String {
         self.symbol.get_or_default()
     }
 
+    /// Returns value 8-bit length of the token.
     pub fn decimals(&self) -> u8 {
         self.decimals.get_or_default()
     }
 
+    /// Returns the amount of all tokens ever minted.
     pub fn total_supply(&self) -> U256 {
         self.total_supply.get_or_default()
     }
 
+    /// Returns balance of tokens for every user.
```

Comment:
```suggestion
    /// Returns the balance of tokens for every user.
```
Above there is "the amount", here is just "balance"? You need to be consistent.

Reply:
Fixed. I thought it'd be good idea to use name of the function in the comment.